src/apps/backend/app.ts

```diff
@@ -42,6 +42,21 @@ export default class App {
     return Promise.resolve(server);
   }
 
+  public static getAllServers(): (
+    | AccountServer
```

Comment:
This can quickly get out of hand, can the return type be array of ApplicationServer?

Reply:
If the path thing keeps failing, you can update the implementation here 

`public static getAPIMicroservices(): APIMicroserviceService[] {}`

Where:

```
type APIMicroserviceService {
  serviceName: string;
  serviceRootFolderName: string;
  serverInstance: ApplicationServer
}
```